PrebidMobile/Logging/Log.swift

```diff
@@ -29,35 +29,46 @@ public class Log: NSObject {
         return formatter
     }
     
+    public static func setCustomLogger(customLogger: PrebidLogger?) {
+        logger = customLogger
+    }
+    
     public static var logLevel: LogLevel = .debug
     public static var logToFile = false
 
     public static func error(_ object: Any, filename: String = #file, line: Int = #line, function: String = #function) {
         log(object, logLevel: .error, filename: filename, line: line, function: function)
```

Comment:
this code should be in the respected function of the `SDKConsoleLogger`. 

Just move it to `error(...)` function in this case.

Reply:
You should move 

```
 log(object, logLevel: .error, filename: filename, line: line, function: function)
```

into  `SDKConsoleLogger.error(_ object: Any, filename: String, line: Int, function: String)` function. 

And the same for all other functions. In this case the SDK's logging functionality will be encapsulated in `SDKConsoleLogger`. And published can replace it with a custom one.